content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,18 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#alpha-kubernetes-io-provided-node-ip}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "constrain-with-nodes"`
+
+Used on: Node
+
+Node affinity and [nodeSelector](https://kubernetes.io/docs/reference/labels-annotations-taints/#schedulerkubernetesnode-selector) are conceptually similar, both enabling you to restrict the scheduling of your Pod to specific nodes in the cluster, based on node labels.
```

Comment:
We prefer to use site-relative links whenever feasible.
```suggestion
Node affinity and [nodeSelector](/docs/reference/labels-annotations-taints/#schedulerkubernetesnode-selector) are conceptually similar, both enabling you to restrict the scheduling of your Pod to specific nodes in the cluster, based on node labels.
```

Reply:
Hey @dipesh-rawat, I added the address you gave but it's giving me `page not found`.
So here's what I did I added `https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/_index.md#scheduleralphakubernetesionode-selector-schedulerkubernetesnode-selector` for now.
So could you assist me with this page not found issue with the address you gave?

Reply:
This link works fine for me, can you recheck it?
This won't work in IDE and has to be tested after running the website locally using `make serve`

Reply:
> This link works fine for me, can you recheck it? This won't work in IDE and has to be tested after running the website locally using `make serve`

Yaa @Ritikaa96,  after checking it locally the address doesn't work so for now I have added the link!